src/main/java/ch/njol/skript/bukkitutil/SoundUtils.java

```diff
@@ -0,0 +1,62 @@
+package ch.njol.skript.bukkitutil;
+
+import org.bukkit.Keyed;
+import org.bukkit.NamespacedKey;
+import org.bukkit.Sound;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Locale;
+
+public class SoundUtils {
+
+	private static final boolean SOUND_IS_INTERFACE;
+
+	static {
+		try {
+			Class<?> SOUND_CLASS = Class.forName("org.bukkit.Sound");
+			SOUND_IS_INTERFACE = SOUND_CLASS.isInterface();
```

Comment:
I hate to do this to you, but this worked in my PR:
```suggestion
			SOUND_IS_INTERFACE = Sound.class.isInterface();
```
(secretly im enjoying the name of your commits)

Reply:
it doesn't feel right that that should work but i guess it does make sense

Reply:
I felt the same, but tests passed, I was shocked, I felt like it should have failed.